addon/components/frost-select.js

```diff
@@ -364,7 +365,8 @@ export default Component.extend({
       const onInput = this.get('onInput')
 
       if (typeOf(onInput) === 'function') {
-        onInput(filter)
+        const debounceInterval = this.get('debounceInterval')
+        run.debounce(this, onInput, filter, debounceInterval)
```

Comment:
Do we want to use ember-concurrency here or will the ember run automatically clean up as well?